configuration/editors.md

```diff
@@ -0,0 +1,164 @@
+     
+    file ..\*\\.(persist)$ openHAB\sPersistence
+    include openhab-persist.syntax
+     
+    file ..\*\\.(rules)$ openHAB\sRules
+    include openhab-rules.syntax 
+```
+- edit the Debian-line from
+`file (rules|rocks)$ Debian\srules`
+to 
+`file (rocks)$ Debian\srules`
+because it interferes with openHABs rules-files.
+
+### Screenshots
+![Items](http://wiki.openhab-samples.googlecode.com/hg/screenshots/syntaxhl_mc_items.png "Items")
+![Rules](http://wiki.openhab-samples.googlecode.com/hg/screenshots/syntaxhl_mc_rules.png "Rules")
+![Sitemap](http://wiki.openhab-samples.googlecode.com/hg/screenshots/syntaxhl_mc_sitemap.png "Sitemap")
+
+## Notepad++
+
+Notepad++ Version 6.2 or above is required to support UDL2 (User Defined Language v2).
+# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"
+http://notepad-plus-plus.org/news/notepad-6.2-release-udl2.html
+
+### How to import UDL2-files?
+
+- Download the UDL2-Files (openHAB-`*`.xml)
+- https://code.google.com/p/openhab-samples/source/browse/syntaxhl/npp/?repo=wiki
+- Install or update Notepad++ if necessary
+- http://notepad-plus-plus.org/download/
+- Open Notepad++
+- Click "Language" (1)
+- Click "Define your language.." (2)
+- Click "Import..." (3)
+- Select one of the downloaded XML-files
+- Done.
+
+![Import](http://wiki.openhab-samples.googlecode.com/hg/screenshots/syntaxhl_npp_import_udl2.png "Import")
+
+### Screenshots
+
+![Items](http://wiki.openhab-samples.googlecode.com/hg/screenshots/syntaxhl_npp_items.png "Items")
+![Rules](http://wiki.openhab-samples.googlecode.com/hg/screenshots/syntaxhl_npp_rules.png "Rules")
+![Sitemap](http://wiki.openhab-samples.googlecode.com/hg/screenshots/syntaxhl_npp_sitemap.png "Sitemap")
```

Comment:
Dead URLs.
We might want to move them to this repository from this thread: https://groups.google.com/forum/#!topic/openhab/VXdBiaY8XB8